com.unity.ml-agents/Runtime/Actuators/ActionSpec.cs

```diff
@@ -66,10 +66,17 @@ public static ActionSpec MakeDiscrete(params int[] branchSizes)
             return actuatorSpace;
         }
 
-        internal ActionSpec(int numContinuousActions, int[] branchSizes = null)
+        /// <summary>
+        /// Create an ActionSpec initialized with the specified action sizes.
```

Comment:
I would like to propose adding "Writing docstrings for constructors" to the list of "Hard things in Computer Science".